src/app/centers/centers.module.ts

```diff
@@ -20,7 +21,7 @@ import { CentersService } from './centers.service';
     FlexLayoutModule
 
   ],
-  declarations: [CentersComponent],
+  declarations: [CentersComponent, CreateCenterComponent],
```

Comment:
You can add the component in the next line, as done in other modules. And if you don't mind can you document this file as it will be helpful later.